### PR TITLE
MAT-5211 update useCqlLibraryServiceApi with APIs used by madie-cql-l…

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
…ibrary

## MADiE PR

Jira Ticket: [MAT-5211](https://jira.cms.gov/browse/MAT-5211)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by jkotanchik-SB — APPROVED** [review]

**Review by adongare — APPROVED** [review]

File: src/api/useCqlLibraryServiceApi.ts [hunk 1]
```diff
@@ -295,6 +336,32 @@ export class CqlLibraryServiceApi {
     }
   }
 
+  async transferLibraries(
+    libraryIds: Array<string>,
+    harpId: string,
+    retainShareAccess: boolean
+  ): Promise<any> {
```
Comment (RohitKandimalla) [inline src/api/useCqlLibraryServiceApi.ts:343]: Can we type this response type to List of Strings?

Comment (sb-cecilialiu) [reply src/api/useCqlLibraryServiceApi.ts:343]: @RohitKandimalla The response will be a list of libraryIds. The front end need to handle the response based on status: 
transferLibraries
